legacy/src/test/java/org/opensearch/sql/legacy/unittest/utils/PrettyFormatterTest.java

```diff
@@ -31,11 +31,13 @@ public void assertFormatterWithoutContentInside() throws IOException {
     public void assertFormatterOutputsPrettyJson() throws IOException {
         String explainFormattedPrettyFilePath = TestUtils.getResourceFilePath(
                 "/src/test/resources/expectedOutput/explain_format_pretty.json");
-        String explainFormattedPretty = Files.toString(new File(explainFormattedPrettyFilePath), StandardCharsets.UTF_8);
+        String explainFormattedPretty = Files.toString(new File(explainFormattedPrettyFilePath), StandardCharsets.UTF_8)
+                .replaceAll("\r", "");
 
         String explainFormattedOnelineFilePath = TestUtils.getResourceFilePath(
                 "/src/test/resources/explain_format_oneline.json");
-        String explainFormattedOneline = Files.toString(new File(explainFormattedOnelineFilePath), StandardCharsets.UTF_8);
+        String explainFormattedOneline = Files.toString(new File(explainFormattedOnelineFilePath), StandardCharsets.UTF_8)
+                .replaceAll("\r", "");
```

Comment:
this second replace might be unnecessary

but i didn't really understand why removing `\r` in `explainFormattedPretty` fixes the test. 

1. json file stored in index doesn't have `\r`
2. windows git checkouts the file to disk
3. java reads file into a string using `Files.toString()`

when is the `\r` added?

Reply:
nvm i see why, it's `assertThat(actual, expected)` not `assertThat(expected, actual)`